README.md

```diff
@@ -347,3 +347,25 @@ The `beaver dlang make` command also uses some utilities that could come handy
 if you want to write a custom build script. Take a look at the `lib/dlang.sh`
 file, you'll find the utility functions with documentation in there.
 
+
+### Codecov support
+
+Beaver includes a convenient wrapper to send [codecov.io](https://codecov.io/)
+reports. Just make sure the project was already built with coverage support and
+then use something like this in the `.travis.yml` file:
+
+```yml
+after_success: beaver dlang codecov [OPTIONS]
+```
+
+The `[OPTIONS]` are forwarded directly to the
+[codecov-bash](https://github.com/codecov/codecov-bash), but this script will
+run in a confined docker instance where only the coverage reports are available
+(this means no file list will be sent to codecov).
+
+Only `TRAVIS*`, `CODECOV_*` and some dlang-related (`DIST DMD DC F V` etc.)
+environment variables will be, if you want to pass any other environment
+variables use `BEAVER_DOCKER_VARS` as usual.
+
+The following environment variables are reported to codecov (via the `-e`
```

Comment:
> The following environment variables are automatically (via the `-e` option) reported to code cov: `DIST`, `DMD`, `DC`, `F`.

Reply:
Mmm, I don't like it this way, take a look at my new take and let me know what you think.

Reply:
Yeah, I think it's fine.